src/main/java/fr/insee/rmes/model/organisation/OrganisationModel.java

```diff
@@ -41,19 +66,67 @@ public void setId(String id) {
         this.id = id;
     }
 
-    public String getLabelLg2() {
-        return labelLg2;
+    public String getUri() {
+        return uri;
+    }
+
+    public void setUri(String uri) {
+        this.uri = uri;
+    }
+
+    public String getAbreviation() {
+        return abreviation;
+    }
+
+    public void setAbreviation(String abreviation) {
+        this.abreviation = abreviation;
+    }
+
+    public String getUniteDe() {
+        return uniteDe;
+    }
+
+    public void setUniteDe(String uniteDe) {
+        uniteDe = uniteDe;
+    }
+
+    public String getSousTelleDe() {
```

Comment:
Is it an english word ?

Reply:
No a french one. In the spec they are asking for a "SousTelleDe" property which mean "under the supervision of "